src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -332,6 +332,53 @@ public Matrix Transform
             }
         }
 
+        /// <summary>
+        /// Gets or sets the world transform elements for this <see cref="Graphics"/>.
+        /// </summary>
+        /// <remarks>
+        /// This is a more performant alternative to <see cref="Transform"/> that does not need disposal.
+        /// </remarks>
+        public unsafe Matrix3x2 TransformElements
+        {
+            get
+            {
+                Gdip.CheckStatus(Gdip.GdipCreateMatrix(out IntPtr nativeMatrix));
+                Gdip.CheckStatus(Gdip.GdipGetWorldTransform(
+                    new HandleRef(this, NativeGraphics), new HandleRef(null, nativeMatrix)));
```

Comment:
This doesn't need to be inside the try?  Seems like if this fails, we won't clean up nativeMatrix?

Reply:
You're right, thanks!